code/__DEFINES/math.dm

```diff
@@ -236,3 +236,6 @@
 		return round(num--)
 	else
 		return num
+
+/// Calculate the angle between two atoms
+#define ATOM_ANGLE(ATOM_A, ATOM_B) (180 + arctan((ATOM_A.y - ATOM_B.y), (ATOM_A.x - ATOM_B.x)))
```

Comment:
`Get_Angle` already exists which does roughly the same. It however also keeps track of the atoms pixel shifts.
Prob best to make use of that one to avoid dupe code

Reply:
Why the hell do we have two of these which have the same name and do the same thing, except one doesnt account for pixel shifts 
![image](https://user-images.githubusercontent.com/25063394/123254828-0ae13f80-d4e7-11eb-819f-88058b975ff8.png)